pegasus/src/homepage.rb

```diff
@@ -0,0 +1,130 @@
+class Homepage
+  def self.get_heroes
+    [
+      {text: "homepage_hero_text_stat_loc",      centering: "50% 40%", type: "stat",    textposition: "bottom", image: "/images/homepage/girl2.jpg"},
+      {text: "homepage_hero_text_stat_students", centering: "50% 30%", type: "stat",    textposition: "bottom", image: "/images/homepage/girl6.jpg"},
+      {text: "homepage_hero_text_stat_served",   centering: "50% 35%", type: "stat",    textposition: "top",    image: "/images/homepage/girl9.jpg"},
+      {text: "homepage_hero_text_malala",        centering: "50% 30%", type: "celeb",   textposition: "bottom", image: "/images/homepage/malala.jpg"},
+      {text: "homepage_hero_text_susan",         centering: "52% 6%",  type: "celeb",   textposition: "top",    image: "/images/homepage/susan.jpg"},
+      {text: "homepage_hero_text_satya_new",     centering: "47% 21%", type: "celeba",  textposition: "bottom", image: "/images/homepage/satya_new.jpg"},
+      {text: "homepage_hero_text_sheryl",        centering: "59% 22%", type: "celeb",   textposition: "bottom", image: "/images/homepage/sheryl.jpg"},
+      {text: "homepage_hero_text_student1",      centering: "50% 30%", type: "student", textposition: "bottom", image: "/images/homepage/girl.jpg"},
+      {text: "homepage_hero_text_student2",      centering: "55% 7%",  type: "student", textposition: "top",    image: "/images/homepage/boy5.jpg"},
+      {text: "homepage_hero_text_bosh",          centering: "62% 33%", type: "celeb",   textposition: "bottom", image: "/images/homepage/boy7.jpg"},
+      {text: "homepage_hero_text_student4",      centering: "41% 20%", type: "student", textposition: "bottom", image: "/images/homepage/kids4.jpg"},
+      {text: "homepage_hero_text_student5",      centering: "45% 31%", type: "student", textposition: "top",    image: "/images/homepage/boy3.jpg"},
+      {text: "homepage_hero_text_teacher1",      centering: "24% 11%", type: "teacher", textposition: "bottom", image: "/images/homepage/girl10.jpg"},
+      {text: "homepage_hero_text_teacher2",      centering: "49% 17%", type: "teacher", textposition: "bottom", image: "/images/homepage/kids10.jpg"}
+    ]
+  end
+
+  def self.get_actions
+    [
+      {text: "homepage_action_text_learn",     type: "cta_button", url: CDO.studio_url("/courses")},
+      {text: "homepage_action_text_codevideo", type: "video",      youtube_id: "nKIu9yen5nc", download_path: "//videos.code.org/social/what-most-schools-dont-teach.mp4", facebook: "https://www.facebook.com/Code.org/videos/10100689712053311/", twitter: "Anybody can learn computer science, starting with an #HourOfCode. https://twitter.com/codeorg/status/828716370053304321"}
```

Comment:
Nit: maybe more readable to put each key on its own line.

Reply:
agreed, the lines are getting looooooong.